.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: '🐛 Bug report'
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in `whatsapp-web.js` (https://github.com/pedroslopez/whatsapp-web.js).
+        If you have a question about how to achieve something and are struggling, please post a question
```

Comment:
@PurpShell @pedroslopez - you might need to enable github discussions for your repository for the discussions tab
- https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/enabling-or-disabling-github-discussions-for-a-repository

Reply:
i don't think we should do that, again.

Reply:
Sounds good 👌